File: DataSummitSolution/DataSummitUI/src/app/efModels/aspNetUserClaim.ts
import { AspNetUser } from "../account/models/asp-net-user";

export class AspNetUserClaim {

	Id: number;
	ClaimType: string;
	ClaimValue: string;
	UserId: string;
	User: AspNetUser;

	constructor(id?: number, claimType?: string, claimValue?: string,
		userId?: string, user?: AspNetUser)
	{
		this.Id = id || 0;
		this.ClaimType = claimType || "";
		this.ClaimValue = claimValue || "";
		this.UserId = userId || "";
		this.User = user;
	}
}

File: DataSummitSolution/DataSummitModels/DB/AspNetUserClaim.cs
﻿using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class AspNetUserClaim
    {
        public int Id { get; set; }
        public string ClaimType { get; set; }
        public string ClaimValue { get; set; }
        public string UserId { get; set; }

        public virtual AspNetUser User { get; set; }
    }
}
